### PR TITLE
Update phpstan to 0.10.8

### PR DESCRIPTION
**Pull request by akondas**
Also changed `level` to `max` in configuration.

### CODE REVIEW

File: lib/Doctrine/Inflector/Inflector.php [hunk 1]
```diff
@@ -37,7 +37,7 @@ public function __construct(WordInflector $singularizer, WordInflector $pluraliz
      */
     public function tableize(string $word) : string
     {
-        return mb_strtolower(preg_replace('~(?<=\\w)([A-Z])~u', '_$1', $word));
+        return mb_strtolower((string) preg_replace('~(?<=\\w)([A-Z])~u', '_$1', $word));
```
Comment (Majkl578) [inline lib/Doctrine/Inflector/Inflector.php]: These changes are not valid, it hides the error instead of reacting on it.

Comment (jwage) [reply lib/Doctrine/Inflector/Inflector.php]: https://github.com/doctrine/inflector/issues/115

Comment (jwage) [reply lib/Doctrine/Inflector/Inflector.php]: I started to throw exceptions when preg_replace returns null but it got so complicated so quick in some areas where we chain multiple preg_replace calls together. I'll work on it in #115

Comment (jwage) [reply lib/Doctrine/Inflector/Inflector.php]: Also, do you know in what scenarios preg_replace would return null?

Comment (Majkl578) [reply lib/Doctrine/Inflector/Inflector.php]: https://github.com/phpstan/phpstan/pull/1399#issuecomment-441074705

Comment (akondas) [reply lib/Doctrine/Inflector/Inflector.php]: There is also https://github.com/thecodingmachine/safe

**Pull request merged**